rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/db/RMDBManager.java

```diff
@@ -373,6 +381,8 @@ public Void doInTransaction(Session session) {
         });
     }
 
+    //======================================================================================
```

Comment:
It could be good to explain why there a separation in the code here

Reply:
Indeed it was for self-readability but I can remove it